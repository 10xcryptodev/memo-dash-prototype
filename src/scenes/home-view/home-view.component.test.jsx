import React from 'react'
import { shallow } from 'enzyme'
import HomeViewComponent from './home-view.component'

describe('<HomeViewComponent />', () => {
  let wrapper

  beforeEach(() => {
    const div = document.createElement('div')
    document.body.appendChild(div)
    wrapper = shallow(<HomeViewComponent getMemos={jest.fn()} />)
  })

  it('renders without crashing', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
