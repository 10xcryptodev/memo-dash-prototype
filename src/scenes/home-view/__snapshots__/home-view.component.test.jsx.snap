// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeViewComponent /> renders without crashing 1`] = `
<Fragment>
  <Container
    style={
      Object {
        "marginTop": "7em",
      }
    }
    text={true}
  >
    <Connect(WithFormik(PostMemoComponent)) />
    <Connect(MemosComponent) />
  </Container>
</Fragment>
`;
